Sources/Utilities/KMZendeskChatHandler.swift

```diff
@@ -58,71 +77,131 @@ public class KMZendeskChatHandler {
         self.userId = id
     }
     
+    public func handedOffToAgent(groupId: String) {
+        self.groupId = groupId
+        isHandOffHappened = true
+        sendChatInfo()
+        sendChatTranscript()
+    }
+
+    
     public func getGroupId() -> String {
         return groupId
     }
+    public func updategroupId(_ groupId: String) {
```

Comment:
From the name, it's not clear what's happening.

Reply:
updated the function name